### ISSUE TITLE
PPO2 Atari cannot reproduce paper results

### ISSUE THREAD

**Issue by llach** [openai/baselines#876]
I'm trying to reproduce the reported PPO2 results on the Atari environments.
My script looks like this:

```python
from baselines.run import main

args = [
    '--env', 'BreakoutNoFrameskip-v4',
    '--alg', 'ppo2',
    '--num_env', '16',
    '--log_interval', '1',
    '--seed', '0',
]

main(args)
```

It basically uses the default Atari parameters for PPO which match those from the paper (in `ppo2/defaults.py`), but after 10M steps, my final reward was ~20 with a peak at 30. This doesn't come close to reported performance which was ~300. 
What am I missing here?

**Comment by abhinavsagar** [comment]
It would be more useful if you could provide more information as well as log files. Without further information it's like guessing of the possible problems.

**Comment by DanielTakeshi** [comment]
I ran

```
python -m baselines.run --alg=ppo2 --env=BreakoutNoFrameskip-v4 --network=cnn --num_timesteps=1e7 --ent_coef=0.01 --num_env=8
```

On master and got to 400+ points at the end.

**Comment by llach** [comment]
@abhinavsagar I realise that my issue is not very detailed. I was hoping that someone either observed similar learning behaviour or reports a successful training that confirmed the paper results so I can compare hyperparameter.

@DanielTakeshi I'll start the training today and will report the results once it is finished.

**Comment by llach** [comment]
I've ran several trainings on the current master and all achieved rewards between 240 and 400. During these tests, I could not reproduce my initial training that failed to learn.

Anyway, as the results are satisfactory I suspect I had something messed up in my previous setup. Next time, I'll run the trainings on a clean environment before posting an issue. Thanks for your support!
